openapi/components/schemas/CreditMemos/CreditMemoProperties.yaml

```
@@ -0,0 +1,172 @@
+id:
+  description: Credit memo ID.
+  readOnly: true
+  allOf:
+    - $ref: ../ResourceId.yaml
+number:
+  description: An auto-incrementing number based on the sequence of credit memos for any particular customer.
+  readOnly: true
+  type: integer
+allocations:
+  type: object
+  description: Allocations reduce unused amount of the credit memo.
+  properties:
+    transactions:
+      type: array
+      description: |-
+        List of transactions (typically refunds, credits, and chargebacks) that are allocated to reduce the unused amount of a credit memo.
+      items:
+        type: object
+        properties:
+          transactionId:
+            description: ID of the transaction that the credit memo is allocated to.
+            allOf:
+              - $ref: ../ResourceId.yaml
+          amount:
+            description: |-
+              Amount of credit that is allocated from the credit memo to the transaction.
+              If the `amount` value is not supplied or exceeds the unused amount of the credit memo or the transaction amount, the lesser of the following two values is used:
+              - The unused amount of the credit memo.
+              - The transaction amount.
+            type: number
+            format: double
+          currency:
+            readOnly: true
+            allOf:
+              - $ref: ../CurrencyCode.yaml
+          createdTime:
+            description: Date and time at which a credit memo is allocated.
+            allOf:
+              - $ref: ../ServerTimestamp.yaml
+          updatedTime:
+            $ref: ../UpdatedTime.yaml
+items:
+  type: array
+  items:
+    type: object
+    required:
+      - unitPrice
+      - quantity
+    properties:
+      id:
+        description: ID of the credit memo item.
+        readOnly: true
+        allOf:
+          - $ref: ../ResourceId.yaml
+      invoiceItemId:
+        description: ID of the invoice item on which the the credit item is referenced.
+        nullable: true
+        allOf:
+          - $ref: ../ResourceId.yaml
+      description:
+        description: Description of the credit memo item.
+        type: string
+      unitPrice:
+        description: Price of the credit memo item.
+        type: number
+        format: double
+      quantity:
+        description: Quantity of the credit memo item.
+        type: integer
+      price:
+        description: Total price of the credit memo item.
+        type: number
+        format: double
+        readOnly: true
+      productId:
+        description: ID of the related product.
+        nullable: true
+        allOf:
+          - $ref: ../ResourceId.yaml
+      planId:
+        description: ID of the related plan ID.
```

issue: possible typo
```suggestion
        description: ID of the related plan.
```